packages/plugin-core/src/utils/ExtensionUtils.ts

```diff
@@ -439,6 +441,64 @@ export class ExtensionUtils {
     if (previewTheme !== undefined) {
       _.set(trackProps, "previewTheme", previewTheme);
     }
+
+    const allExtensions = vscode.extensions.all;
+    let allNonBuiltInExtensions = allExtensions.filter((extension) => {
+      return !extension.packageJSON.isBuiltin;
+    });
+    if (VSCodeUtils.isDevMode()) {
+      // done to make this work during dev mode
+      allNonBuiltInExtensions = allNonBuiltInExtensions.filter((ext) => {
+        return !ext.extensionPath.includes("packages/plugin-core");
+      });
+    }
+    try {
+      const extensionsDetail = allNonBuiltInExtensions.map((extension) => {
+        const { packageJSON } = extension;
+        const { id, version } = packageJSON;
+        return { id, version };
+      });
+      if (extensionsDetail && extensionsDetail.length > 0) {
+        _.set(trackProps, "extensionsDetail", JSON.stringify(extensionsDetail));
+        _.set(trackProps, "numExtensions", extensionsDetail.length);
+      }
+    } catch (error) {
+      // something went wrong don't track extension detail
+    }
+
+    try {
+      // infer install path from non-built-in extension paths
+      const installPaths = allNonBuiltInExtensions.map((ext) => {
```

Comment:
this seems expensive. can we just find the directory based on the directory that the dendron extension is installed form?